src/ec/suite_b/private_key.rs

```diff
@@ -119,6 +119,13 @@ fn private_key_as_scalar_(ops: &PrivateKeyOps, private_key: &ec::PrivateKey)
     Scalar::from_limbs_unchecked(&limbs)
 }
 
+// Are limbs within (0, max_exclusive)?
+fn are_limbs_within_range(limbs: &[Limb], max_exclusive: &[Limb]) -> bool {
```

Comment:
Either this should take a `Scalar` or `PrivateKey` argument or we should find a way to share it between the RSA and ECC code, since I think it is exactly the same, even if we don't share the `Range` stuff.